src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureVirtualNetworkGatewayCommand.cs

```diff
@@ -216,9 +219,25 @@ private PSVirtualNetworkGateway CreateVirtualNetworkGateway()
                 }
             }
 
-            if (this.EnableActiveActiveFeature.IsPresent && !vnetGateway.Sku.Tier.Equals(MNM.VirtualNetworkGatewaySkuTier.HighPerformance))
+            if (this.EnableActiveActiveFeature.IsPresent 
```

Comment:
Tests for these changes?  Also, please update the Changelog.md for Network to reflect the changes